sakip_sabanci_config.rb

```diff
@@ -29,14 +31,14 @@
 to_field 'cho_date_range_norm', extract_oai('dc:date[1]'), strip
 to_field 'cho_description', extract_oai('dc:description'), strip
 to_field 'cho_dc_rights', extract_oai('dc:rights'), strip
-to_field 'cho_edm_type', extract_oai('dc:type'),
-         strip, transform(&:downcase), translation_map('not_found', 'types', 'turkish-types')
+to_field 'cho_edm_type', extract_oai('dc:type'), normalize_type
```

Comment:
Does this change belong in this PR?  It seems unlike the rest of the changes.

Reply:
Good catch, that should be a separate commit. I will fix it after lunch.